certified-connectors/Tilkee-V3/README.md

```diff
@@ -0,0 +1,32 @@
+# Tilkee
+
+Tilkee fuels your business by empowering your Sales and Marketing teams through reliable and actionable data. This connector offers the ability to integrate your other applications with Tilkee.
+

```

Comment:
@FloxoS Please refer the templates provided for readme.md file in the repo and add required sections.